modules/ROOT/pages/try-scope-concept.adoc

```diff
@@ -49,6 +49,90 @@ A new transaction is started every time the scope is executed.
 +
 Only relevant in cases where execution order may vary (for example, due to asynchronous actions occurring outside the flow): If the current processing of the flow has already begun a transaction, join it. Otherwise, begin a new one.
 
+=== Analyzing different Transactional Actions in Try Scope
+
+Supponse we have the next configuration: 
+
+[source,xml,linenums]
+----
+<flow name="someFlow">
+	<jms:listener config-ref="JMS_Config" destination="test.in" transactionalAction="ALWAYS_BEGIN"/>
+	<!-- Processors -->
+	<try transactionalAction="?">
+		<!-- Processors -->
+		<!-- Join if possible is the default value for jms:publish operation --> 
+		<jms:publish config-ref="JMS_Config" destination="test.out"/>
+		<raise-error type="APP:SOME"/>
+		<!-- More Processors -->
+	</try>
+	<!-- Processors -->
+</flow>
+----
+
+As expected, in case there are no processors that produces any error this will finish the execution and commit the transaction, independently of the `transactionalAction` set in the Try Scope. 
+
+We will analyze what happends with the transaction and messages with the `<raise-error/>` processor added, depending on the `transactionalAction` of the Try Scope: 
+
+* Ignore (`INDIFFERENT`): in this case, the transaction is _suspended_ while executing the try scope inner processors. When the error is raised, it is propagated to the source (which does not handle it with an `<on-error-continue>`). So, the transaction is rollbacked, and the message will be available again in the JMS queue. However, the message sent in the `jms:publish` operation has completely been sent, since the rollback of the transaction does not affect the publish operation, which was outside of the transaction scope (even though the `transactionalAction` of the `jms:publish` operation is `BEGIN_OR_JOIN`). This will be the exact same behaviour if, instead of having a Try Scope with a `transactionalAction="INDIFFERENT"`, we had the `jms:publish` with `transactionalAction="NOT_SUPPORTED"`. 
+* Always Begin (`ALWAYS_BEGIN`): this time a new transaction is began for the execution of the try scope. After the error is raised (and not handled), the new transaction is rollbacked, which means that the `jms:publish` operation will not commit and thus the message is not actually sent to the `test.out` queue. Then, the transaction of the flow (which was began by the message source) will also be rollbacked, and message will be available again in the `test.in` queue. 
+* Begin or Join (`BEGIN_OR_JOIN`): in this case, when the error is raised, it is not inmediately rollbacked. Since the flow could handle the raised error (therfore, commit the transaction) the message sent by the `jms:publish` could still be sent. This time, since the error is not handled with an `on-error-continue`, then the entire transaction is rollbacked and the result is the same as in the case of Always Begin. 
```

Comment:
"Since the flow could handle the raised error (therfore, commit the transaction) the message sent by the `jms:publish` could still be sent."

This sentence is not very clear, could you please confirm if the option below would be correct:
"Since the flow might be able to handle the raised error and commit the transaction, the `jms:publish` operation can still complete and send the message"